cmd/apply/cmdapply.go

```diff
@@ -153,7 +153,7 @@ func (r *ApplyRunner) RunE(cmd *cobra.Command, args []string) error {
 		ReconcileTimeout:  r.reconcileTimeout,
 		// If we are not waiting for status, tell the applier to not
 		// emit the events.
-		EmitStatusEvents:       emitStatusEvents,
+		EmitStatusEvents:       printStatusEvents,
```

Comment:
Correct. But I'm not changing that functionality in this PR, since it is already very big. This gives us the same current behavior, but the `PrintStatus` is not hard-coded to the existence of a `WaitTask`. The `EmitStatusEvents` cleanup PR should remove all `EmitStatusEvents` options and checks, and just send the status events no matter what. The biggest change with this cleanup PR will be all the tests which are expecting events in a particular order.